include/clad/Differentiator/CladUtils.h

```diff
@@ -207,6 +208,10 @@ namespace clad {
     /// `{arr, arrSize}`
     clang::Expr* BuildCladArrayInitByConstArray(clang::Sema& semaRef,
                                                 clang::Expr* constArrE);
+
+    /// Returns true if `FD` is a class instance method; otherwise returns
+    /// false.
```

Comment:
```suggestion
    /// Returns true if `FD` is a class member function; otherwise returns
    /// false.
```

Perhaps adjust the invert the name of the function to `IsStaticMethod` as well to somewhat clearer to the casual reader.